docusaurus/docs/how-to-guides/data-source-plugins/add-support-for-externally-shared-dashboards.md

```diff
@@ -0,0 +1,63 @@
+---
+id: add-support-for-externally-shared-dashboards
+title: Add support for externally shared dashboards
+description: How to add support for externally share dashboards (previously called Public dashboards).
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - externally shared dashboards
+  - public dashboards
+  - data source
+  - datasource
+---
+
+Externally shared dashboards (previously called Public dashboards) retrieve the query from the backend, instead of receiving it from the frontend. That is due to security purposes, to not expose sensitive data and also not be able to perform unauthorized queries.
+
+Because of that, it is necessary to not pass any frontend-transformed body to the request, as it will not be used in the externally shared dashboard panel request.
```

Comment:
Do you mean that frontend-transformed bodies are automatically not passed to the request or that users need to avoid doing this?

```suggestion
Because of this, it's necessary to not pass any frontend-transformed body to the request, as it won't be used in the externally shared dashboard panel request.
```